wagtailio/templates/base.html

```diff
@@ -60,92 +59,74 @@
             <!-- End Google Tag Manager -->
         {% endif %}
 
-        {% compress css %}
-            <link rel="stylesheet" type="text/x-scss" href="{% static 'css/main.scss' %}" />
-        {% endcompress %}
+        <link rel="stylesheet" href="{% manifest 'css/main.css' %}">
 
         {% block extra_css %}{% endblock %}
 
     </head>
     <body class="{% if page %}template-{{ page.get_verbose_name|slugify }}{% endif %} {% block body_class %}{% endblock %} ">
-        {% if not DEBUG %}
-            <!-- Google Tag Manager (noscript) -->
-            <noscript>
-                <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5XDDLH"
-                height="0" width="0" style="display:none;visibility:hidden"></iframe>
-            </noscript>
-            <!-- End Google Tag Manager (noscript) -->
-        {% endif %}
-
-        {% if FB_APP_ID %}
-        <div id="fb-root"></div>
-
-        <script>
-            window.fbAsyncInit = function() {
-                FB.init({appId: {{ FB_APP_ID }}, xfbml: true, version: 'v2.1'});
-            };
-
-            (function(d, s, id){
-            var js, fjs = d.getElementsByTagName(s)[0];
-            if (d.getElementById(id)) {return;}
-            js = d.createElement(s); js.id = id;
-            js.src = "https://connect.facebook.net/en_US/sdk.js";
-            fjs.parentNode.insertBefore(js, fjs);
-            }(document, 'script', 'facebook-jssdk'));
-        </script>
-        {% endif %}
-        {% wagtail_ab_testing_script %}
+        {% include "includes/svg_sprites.html" %}
```

Comment:
Question: What's this?

Reply:
It's a sprite sheet that already exists - https://github.com/chris-lawton/wagtail.org/blob/add-tooling-remove-compressor/wagtailio/templates/includes/svg_sprites.html. We discussed using separate html templates for the icons that the editor will be able to choose from. I'm wondering if it would be useful to include icons in that won't be in that list but will be used throughout the site.

Reply:
Fair enough, i see your point.